src/dispatch/case/flows.py

```diff
@@ -73,46 +186,36 @@ def case_new_create_flow(*, case_id: int, organization_slug: OrganizationSlug, d
     case = get(db_session=db_session, case_id=case_id)
 
     # we create the ticket
-    ticket = ticket_flows.create_case_ticket(case=case, db_session=db_session)
+    ticket_flows.create_case_ticket(case=case, db_session=db_session)
 
-    if not ticket:
-        # we delete the case
-        delete(db_session=db_session, case_id=case_id)
+    individual_participants, team_participants = get_case_participants(
+        case=case, db_session=db_session
+    )
 
-        return
+    # TODO add reporter
+    participant_emails = [case.assignee.email]
```

Comment:
This should change to `case.assignee.individual.email` in https://github.com/Netflix/dispatch/pull/2800 depending on merge order. Git should catch, just a heads up.